pkg/scheduler/framework/plugins/nodename/node_name.go

```diff
@@ -37,6 +38,14 @@ const (
 	ErrReason = "node(s) didn't match the requested node name"
 )
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *NodeName) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Node, ActionType: framework.Add | framework.UpdateNodeLabel},
```

Comment:
does updating the name label really change the node's name? this [1] says that you need to make changes to the kubelet config so that it rejoins the cluster, which is covered by framework.Add.

[1] https://stackoverflow.com/questions/46006716/how-to-change-name-of-a-kubernetes-node

Reply:
I don't quite think updating a node's label includes its name.

Reply:
Yes! You are all right. Did a quick experiment: 

1. added a breakpoint to nodename code
2. update the node label 
3. nothing triggerred. 

Conclusion: framework.UpdateNodeLabel shouldn't be here.